Rethinking_2/Chp_06.ipynb

```diff
@@ -0,0 +1,3109 @@
+{
```

Comment:
<p>It would be better if you could define a helper function at the top of the NB, and then call it here, with a loop or else.</p><p>Something like:</p><pre class="ql-syntax" spellcheck="false">def standardize(series):
    """Standardize a pandas series"""
    return (series - series.mean()) / series.std()
</pre><p>Also, you don't need the <code>.loc</code>  operator when selecting pandas columns: <code>d["K"]</code>  will do.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>